planet/src/main/webapp/js/authentication.js

```diff
@@ -111,7 +111,7 @@ async function signUp() {
     firebase.auth().createUserWithEmailAndPassword(email, password)
     .then(function() {
         const user = firebase.auth().currentUser;
-        addUserToDatabase(user.uid, displayName, email, phoneNumber, username);
+        addUserToDatabase(user.uid, displayName, phoneNumber, email, username);
```

Comment:
changes like these are not exactly relevant to the PR's title. I understand that it feels natural to group small changes into a PR with other changes but please keep them separate.
Having small, bug specific PRs makes reviewing easier and faster.

Reply:
Okay! I will add this fix in another PR.